### PR TITLE
ADCM-1571 add allure steps

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+max-line-length=100
```
Comment (vgorkavenko) [inline .flake8:3]: Better `ignore = E501` Then only pylint will check length of string

File: tests/functional/test_import.py [hunk 2]
```diff
@@ -28,15 +17,19 @@ def test_service_import_negative(sdk_client_fs: ADCMClient, path):
...
+    with allure.step('Create cluster with def import'):
+        bundle_import = sdk_client_fs.upload_from_fs(path + '/import')
+        cluster_import = bundle_import.cluster_create("cluster_import")
+    with allure.step('Bind service from cluster with export to cluster with import'):
+        cluster_import.bind(cluster)
     with pytest.raises(coreapi.exceptions.ErrorMessage) as e:
         cluster_import.bind(service)
-    err.BIND_ERROR.equal(e)
+    with allure.step('Expect backend error because incorrect version for import'):
+        err.BIND_ERROR.equal(e)
```
Comment (vgorkavenko) [inline tests/functional/test_import.py:32]:
```suggestion
        assert err.BIND_ERROR == e
```

Comment (vgorkavenko) [reply tests/functional/test_import.py:32]: View in all similar places

File: .flake8 (hunk 1 above)
Comment (annademina) [reply .flake8]: fixed

File: tests/functional/test_import.py (hunk 2 above)
Comment (annademina) [reply tests/functional/test_import.py]: https://arenadata.atlassian.net/browse/ADCM-1581

Comment (vgorkavenko) [reply tests/functional/test_import.py]: Pls revert. Didn't know it was a custom `equal`

**Review by skhomuti — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**